### ISSUE TITLE
Allo Digione Signature + Volumio + Kodi plugin = No sound

### ISSUE THREAD

**Issue by adriano-art** [Saiyato/volumio-kodi-plugin#29]
Hi everyone!
I'm newbie about Volumio &Co. I've buyed Allo Digione Sig. player, and I've installed Kodi plugin.
I need some help, because I can't hear anything by kodi player, but music playing on volumio soft sounds great.
How can I fix this problem? Please help me..

Thank you!

PS: I set ALSA output on kodi and disabled sturtup sound on volumio too. What else have me to do?
:? 

Thanks.

**Comment by Saiyato** [comment]
Hi there,

Have you followed the steps in the readme? You can find it here: https://github.com/Saiyato/volumio-kodi-plugin

**Comment by adriano-art** [comment]
Thank you for your reply!
I've read that "readme" and was helpful to install IPTV PVR client.. Thanx for your work!

Unfortunately, I can't play any sound from my Coaxial output of Allo Digione Signature player.

Option:
- HDMI Hotplug: i need this to on?
- Enable DAC Output for Kodi:    ON
- Configure Kali Sound delay:     ON .  (0,7)
- Kodi GUI SOUND:     ON
- Have KODI Keep Sound device alive:     ON

Is that all right? There's something wrong? 
What can I do to ear by Allo Digione on Kodi?? Help!!! 

:-))

**Comment by adriano-art** [comment]
Teen, other options in kodi plugin, audio menu:
- audio output: ALSA: DEFAULT (BCM2835)
- OUTPUT config: optimized;
- reclock quality: high
- keep sound device alive: ALWAYS

Other options available As default.

Thank you for all!

Adriano

**Comment by adriano-art** [comment]
Hello! 
Any suggest?

Thank you,
Adriano

**Comment by Saiyato** [comment]
Hi @adriano-art , you need to select your soundcard in Kodi, not ALSA if you want the sound to be played over the DigiOne, that should solve your problem.

**Comment by Aaike1955** [comment]
Same problem here. Allodigione soundcard does not show as an option in Kodi. Tried everything and settings in Volumio are okay. What to do ?

**Comment by Saiyato** [comment]
Did you change your volumio password by any chance? I see there's some legacy code in the script, that might cause this behavior

**Comment by Aaike1955** [comment]
Thx for reply.
Yes you are right I had two accounts and deleted the oldest.
Signed in with latest account and verified that old one is deleted.
No result Allodigione Alsa SPDIF does not show as option in Kodi.
Any other suggestions ?

**Comment by Saiyato** [comment]
Yes, this is because of the new bluetooth implementation, Kodi is not enumerating the correct sinks.
Quick and VERY dirty workaround is to uninstall pulseaudio and change the index of the default card to the correct value (possibly 2).

Check the index with aplay -L
```
card 2: sndrpihifiberry [snd_rpi_hifiberry_digi], device 0: HifiBerry Digi HiFi wm8804-spdif-0 [HifiBerry Digi HiFi wm8804-spdif-0]
  Subdevices: 1/1
  Subdevice #0: subdevice #0
```
In my case the hifiberry has index 2 (card 2). So I had to change /etc/asound.conf to:
```
#KODI
        defaults.ctl.card 2
        defaults.pcm.card 2
#ENDOFKODI
```
